content/engine/api/_index.md

```diff
@@ -125,7 +125,38 @@ You can specify the API version to use in any of the following ways:
 
 ### API version matrix
 
-{{< include "api-version-matrix.md" >}}
+| Docker version | Maximum API version        | Change log                                               |
+|:---------------|:---------------------------|:---------------------------------------------------------|
+| 25.0           | [1.44](/engine/api/v1.44/) | [changes](/engine/api/version-history/#v144-api-changes) |
```

Comment:
Whoops; looks like I copy/pasta'd my version that already had v25, but the link to the API docs is broken. Wondering why CI didn't fail on this; any ideas, @dvdksn ?

Reply:
Seems we're ignoring these links: https://github.com/docker/docs/blob/main/.htmltest.yml#L11

Reply:
For no good reason I think. I disabled this setting and found two more broken links.

```
=> ERROR [test 5/5] RUN htmltest                                                                                                                                                                                      5.7s 
------                                                                                                                                                                                                                      
 > [test 5/5] RUN htmltest:                                                                                                                                                                                                 
0.139 htmltest started at 10:25:40 on public                                                                                                                                                                                
0.139 ========================================================================                                                                                                                                              
2.947 engine/swarm/manage-nodes/index.html                                                                                                                                                                                  
2.947   hash does not exist --- engine/swarm/manage-nodes/index.html --> /engine/api/v1.31/#operation/ServiceCreate                                                                                                         
2.947 engine/swarm/admin_guide/index.html
3.004   hash does not exist --- engine/swarm/admin_guide/index.html --> /engine/api/v1.25/#tag/Node
3.004 ========================================================================
5.634 ✘✘✘ failed in 5.495239794s
5.634 2 errors in 2096 documents
5.634 
------
```

This was a carry-over from when we were using htmlproofer. Maybe that tool had issues with these?

Reply:
hmm looks like this tool can't figure out these links either, even if I use the "correct" path. I guess it's because the subpaths work on this api doc page. Those aren't actual routes, but I guess they're intercepted somehow by the redocly script that runs on that page.

Reply:
Hm.. interesting

Guess it's fine for now; it will be added soon